### PR TITLE
Bump VS Editor to 15.8

### PR DESCRIPTION
**Pull request by KirillOsenkov**
Update VS Editor to 15.8.

Delete unused QuickInfo code (will use from Editor .dll when we switch our Tooltips to VS QuickInfo).

Implement ITextView2 and integrate IMultiSelectionBroker.

### CODE REVIEW

**Review by ToddGrun — APPROVED** [review]

**Comment by KirillOsenkov** [comment]
Looks like the two SyntaxHighlighting tests are a legitimate problem. They pass on Windows, but I'll check on Mac tomorrow. 

Compat checker failure is expected until we bump to the WebTooling addin that builds against this PR.

Xamarin.Forms test failure seems unrelated.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Composition/ExperimentationServiceInternal.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Text.Utilities;
+
+namespace MonoDevelop.Ide.Composition
+{
+	[Export (typeof (IExperimentationServiceInternal))]
+	internal class ExperimentationServiceInternal : IExperimentationServiceInternal
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Composition/ExperimentationServiceInternal.cs]: Maybe add `Editor` as a prefix here, as there are multiple experimentation services.

**Comment by KirillOsenkov** [comment]
Unit-tests are fixed (was a good find, actual bug!) and the companion PR that bumps WebTools to match this PR is: https://github.com/xamarin/md-addins/pull/3485

With that WebTools the compat checker should be satisfied.

**Comment by KirillOsenkov** [comment]
Hmm, md-addins PR 3485 unit-tests all fail MEF composition with:
`Expected 1 export(s) with contract name "Microsoft.VisualStudio.Mac.LanguageServices.Razor.ProjectSystem.DotNetProjectHostFactory" but found 0`

I haven't seen this before, not sure if benign or a real issue. Will investigate tomorrow.

**Review by Therzok — APPROVED** [review]

**Comment by KirillOsenkov** [comment]
I've manually tested the .dmg from this build and it looks great, MEF works fine. I think this is ready. @slluis

**Review by Therzok — APPROVED** [review]

**Comment by KirillOsenkov** [comment]
Still needs @slluis for some reason ;)

**Review by slluis — APPROVED** [review]

**Pull request merged**